gpflow/posterior.py

```diff
@@ -502,9 +508,6 @@ def _conditional_with_precompute(
                 Kfu_Qinv_Kuf = tf.einsum("...ijk->...kij", tmp)  # move diagonal dim to [-3]
         cov = Kff - Kfu_Qinv_Kuf
 
-        if not full_cov and not full_output_cov:
-            cov = tf.linalg.adjoint(cov)
-
```

Comment:
Why did you remove this ? In this case (not full_cov and not full_output_cov) we want the returned "co"variance - actually just the marginal variances - to have shape [N, Q] not [Q, N]?

Reply:
Good catch - this was a mistake. It has now been updated.